src/panels/config/automation/ha-config-automation.ts

```diff
@@ -23,6 +24,13 @@ class HaConfigAutomation extends HassRouterPage {
 
   @property() public automations: AutomationEntity[] = [];
 
+  private _debouncedUpdateAutomations = debounce((pageEl) => {
+    const newAutomations = this._getAutomations(this.hass.states);
+    if (!this._equal(newAutomations, pageEl.automations)) {
```

Comment:
This might be overkill, but saves the data table some updates on busy systems...

Reply:
This is fine.